manifest.json

```diff
@@ -0,0 +1,246 @@
+        "rootDirPrefix": "MathJax-",
+        "targetDirPrefix": "MathJax-",
+        "jsFiles": ["MathJax.js?config=TeX-AMS-MML_HTMLorMML"]
+      },
+      "ngImageCrop": {
+        "version": "0.3.2",
+        "zipPath": "https://github.com/alexk111/ngImgCrop/archive/v0.3.2.zip",
+        "rootDirPrefix": "ngImgCrop-",
+        "targetDirPrefix": "ng-img-crop-",
+        "jsFiles": ["compile/minfied/ng-img-crop.js"],
+        "cssFiles": ["compile/minfied/ng-img-crop.css"]
+      },
+      "select2": {
+        "version": "3.5.1",
+        "zipPath": "https://github.com/ivaynberg/select2/archive/3.5.1.zip",
+        "rootDirPrefix": "select2-",
+        "targetDirPrefix": "select2-",
+        "jsFiles": ["select2.js"],
+        "cssFiles": ["select2.css"],
+        "css_background_image": ["select2.png", "select2x2.png"]
+      },
+      "codemirror": {
+        "version": "3.19.0",
+        "zipPath": "https://github.com/marijnh/CodeMirror/archive/3.19.0.zip",
+        "rootDirPrefix": "CodeMirror-",
+        "targetDirPrefix": "code-mirror-",
+        "cssFiles": ["lib/codemirror.css", "addon/merge/merge.css"]
+      },
+      "ui-codemirror": {
+        "version": "0.1.2",
+        "zipPath": "https://github.com/angular-ui/ui-codemirror/archive/src0.1.2.zip",
+        "rootDirPrefix": "ui-codemirror-src",
+        "targetDirPrefix": "ui-codemirror-"
+      },
+      "midi-Js": {
+        "version": "2ef687b47e5f478f1506b47238f3785d9ea8bd25",
+        "zipPath": "https://github.com/mudcube/MIDI.js/archive/2ef687b47e5f478f1506b47238f3785d9ea8bd25.zip",
+        "rootDirPrefix": "MIDI.js-",
+        "targetDir": "midi-js-2ef687"
+      },
+      "bowerMaterial": {
+        "version": "0.6.0-rc1",
+        "zipPath": "https://github.com/angular/bower-material/archive/v0.6.0-rc1.zip",
+        "rootDirPrefix": "bower-material-",
+        "targetDirPrefix": "bower-material-",
+        "jsFiles": ["angular-material.js"],
+        "cssFiles": ["angular-material.css"]
+      },
+      "ui-Map": {
+        "version": "0.5.0",
+        "zipPath": "https://github.com/angular-ui/ui-map/archive/v0.5.0.zip",
+        "rootDirPrefix": "ui-map-",
+        "targetDirPrefix": "ui-map-"
+      },
+      "ui-Utils": {
+        "version": "0.1.1",
+        "zipPath": "https://github.com/angular-ui/ui-utils/archive/v0.1.1.zip",
+        "rootDirPrefix": "ui-utils-",
+        "targetDirPrefix": "ui-utils-"
+      },
+      "hammerJs": {
+        "version": "2.0.4",
+        "zipPath": "https://github.com/hammerjs/hammer.js/archive/2.0.4.zip",
+        "rootDirPrefix": "hammer.js-",
+        "targetDirPrefix": "hammer-js-"
+      }
+    },
+    "oppiaTools": {
+      "yuiCompressor": {
+        "version": "2.4.8",
+        "filesPath": "https://github.com/yui/yuicompressor/releases/download/v2.4.8",
+        "files": ["yuicompressor-2.4.8.jar"],
+        "targetDirPrefix": "yuicompressor-"
+      }
+    }
+  },
+  "paths": {
```

Comment:
These were defined in the doc we created and were meant to be used during js optimization